typescript/sdk/src/deploy/verify/ContractVerifier.ts

```diff
@@ -125,6 +154,12 @@ export class ContractVerifier extends MultiGeneric<VerificationInput> {
       return;
     }
 
+    if (await this.isAlreadyVerified(chain, input)) {
+      this.logger(`Contract ${input.name} already verified on ${chain}`);
+      await sleep(200);
```

Comment:
why sleep?

Reply:
Added a comment 

`// There is a rate limit of 5 requests per second`